http/middleware.go

```diff
@@ -56,10 +73,14 @@ func RecoverMiddleware(l logger.ServerLogger) func(http.Handler) http.Handler {
 					rw.SetPanicObject(re)
 					rw.WriteHeader(http.StatusInternalServerError)
 
-					l.With(
-						logger.Any("err", re),
-						logger.Any(requestIDLogFieldName, net.RequestIDFromCtx(r.Context())),
-					).Error("panic recover", nil)
+					logAttributes := []slog.Attr{
+						slog.Any(requestIDLogFieldName, net.RequestIDFromCtx(r.Context())),
+						slog.Any("error", re),
+					}
+					if options.enableStackTrace {
+						logAttributes = append(logAttributes, slog.String("trace", string(debug.Stack())))
+					}
+					l.LogAttrs(r.Context(), slog.LevelError, "panic recover", logAttributes...)
```

Comment:
`l.With` accepts `...any`. I feel like it's weird to define `logAttributes` as `[]any` only to be able to use it in `With` method. This approach, although more explicit and verbose, feel correct.

Reply:
You're right. I checked the documentation and slog source code a little bit more and this way is actually the preferred one because of performance implications. I will check the template whether there is a possibility to rewrite logs to this form.

Resources:
https://dev.to/morgenstern2573/slog-zero-dependency-structured-logging-in-go-apf
https://pkg.go.dev/log/slog#hdr-Attrs_and_Values